test/ActiveRecordFindTest.php

```diff
@@ -80,10 +66,10 @@ public function testFindReturnsNull()
         $lawyer = HonestLawyer::last();
         $this->assertNull($lawyer);
 
-        $lawyer = HonestLawyer::find('first', ['name'=>'Abe']);
+        $lawyer = HonestLawyer::where(['name'=>'Abe'])->first();
```

Comment:
Also have:

 $lawyer = HonestLawyer::where(['name'=>'Abe'])->first;

 $lawyer = HonestLawyer::where(['name'=>'Abe'])->first(1);

 $lawyer = HonestLawyer::where(['name'=>'Abe'])->first(2);

and similarly for last()

Reply:
Not sure about that.

```php
$lawyer = HonestLawyer::where(['name'=>'Abe'])->first;
```

Not a thing.

```php
$lawyer = HonestLawyer::where(['name'=>'Abe'])->first(1);
```

Always returns an array (an empty one in this case), not null.